hrms/api/__init__.py

```diff
@@ -770,6 +770,25 @@ def download_salary_slip(name: str):
 	return f"data:{content_type};base64," + base64content.decode("utf-8")
 
 
+@frappe.whitelist()
+def download_expense_claim(name: str):
```

Comment:
This too can be commonified as a generic function
`def download_pdf(doctype: str, docname: str):`